perun-utils/ldapc-scripts/schemas/perun-schema.ldif

```diff
@@ -72,6 +72,7 @@ olcAttributeTypes: {67}( 1.3.6.1.4.1.8057.2.80.73 NAME 'uuid' DESC 'UUID of enti
 olcAttributeTypes: {68}( 1.3.6.1.4.1.8057.2.80.74 NAME 'rpEnsureVoDefinition' DESC 'Short name of VO, which proxy check if the user is member of' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
 olcAttributeTypes: {69}( 1.3.6.1.4.1.8057.2.80.75 NAME 'rpEnableEnsureVoFiltering' DESC 'If set to True, proxy will trigger the ensureVo filter' EQUALITY caseIgnoreMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
 olcAttributeTypes: {70}( 1.3.6.1.4.1.8057.2.80.76 NAME 'rpLoginURL' DESC 'URL where user can log in to the service.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {71}( 1.3.6.1.4.1.8057.2.80.77 NAME 'studentIdentifiers' DESC 'All student identifiers generated by Perun' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
```

Comment:
Please set Equality, Ordering, etc. as it is for schacPersonalUniqueCodes.

Reply:
Done.